### PR TITLE
Interactive plots (6)

### LINKED ISSUE #55: Print software versions
Rather print all versions from all tools used in the Rmarkdown report at the end of the report, e.g. SessionInfo() print at the end of report.

### LINKED ISSUE #160: Fix volcano and enrichmeht plot layout
### Description of the bug

When many contrasts have to be plotted, the multiple volcano plots are positioned weirdly so that the x axis label is obstructed. Additionally, the x ticks often overlap if a large range has to be plotted for x (example: `/home-link/iivow01/git/batchtest/run_rnadeseq_folder.sh`). Also, the enriched pathway plots are very high (y axis) even if no points exist there (probably because of the dotted line with text).

### Command used and terminal output

_No response_

### Relevant files

_No response_

### System information

_No response_

### PR DESCRIPTION
**Pull request by WackerO**
PCA plots and heatmaps are now interactive (no issue, this was a point in trello); also adds a sessioninfo to the report and closes #55. Additionally, volcano and gost plots are now cleaned up a bit, closes #160 

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/rnadeseq/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/rnadeseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: assets/RNAseq_report.Rmd [hunk 1]
```diff
@@ -894,20 +924,23 @@ The original plot can be downloaded [here](./differential_gene_expression/plots/
...
+            ylab(paste0("PC2: ",percentVar[2], "% variance")) +
+            coord_fixed() +
+            theme_bw() +
+            theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank()) +
+            guides(color=guide_legend(title=NULL))
 ggsave(plot = pca, filename = "differential_gene_expression/plots/PCA_plot.pdf", device = "pdf", dpi = 300)
 ggsave(plot = pca, filename = "differential_gene_expression/plots/PCA_plot.svg", device = "svg", dpi = 150)
 ggsave(plot = pca, filename = "differential_gene_expression/plots/PCA_plot.png", device = "png", dpi = 150)
-pca
+pca <- ggplotly(pca, tooltip = "text", width=750, height=400)
```
Comment (ggabernet) [inline assets/RNAseq_report.Rmd:940]: Does it now already include `SampleName`?

File: assets/RNAseq_report.Rmd [hunk 2]
```diff
@@ -917,25 +950,29 @@ cat("Batch effects were observed in the PCA, so batch effects were corrected.
...
-            coord_fixed()
+            coord_fixed() +
+            theme_bw() +
+            theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank()) +
+            guides(color=guide_legend(title=NULL))
 ggsave(plot = pca2, filename = "differential_gene_expression/plots/PCA_batch_corrected_plot.pdf", device = "pdf", dpi=300)
 ggsave(plot = pca2, filename = "differential_gene_expression/plots/PCA_batch_corrected_plot.svg", device = "svg", dpi = 150)
 ggsave(plot = pca2, filename = "differential_gene_expression/plots/PCA_batch_corrected_plot.png", device = "png", dpi = 150)
-pca2
+pca2 <- ggplotly(pca2, tooltip = "text", width=750, height=400)
```
Comment (ggabernet) [inline assets/RNAseq_report.Rmd:972]: same here

File: assets/RNAseq_report.Rmd (hunk 1 above)
Comment (WackerO) [reply assets/RNAseq_report.Rmd:940]: ah yes, I added that

**Comment by WackerO** [comment]
@ggabernet this is ready for another review :) the batch run is again unstable in its 5sums because I changed the container; will do a follow-up mini PR to check the sums

File: assets/RNAseq_report.Rmd [hunk 3]
```diff
@@ -1078,6 +1110,7 @@ if (isProvided(params$path_contrast_list)) {
 }
 if (isProvided(params$path_contrast_pairs)) {
     contrasts <- read.table(path_contrast_pairs, sep="\t", header = T, colClasses = "character")
+    #TODO: Do I need to watch out for batch effect here as well?
```
Comment (ggabernet) [inline assets/RNAseq_report.Rmd:1113]: Solve or remove line 😄

**Review by ggabernet — APPROVED** [review]
Looks good to me!

**Pull request merged**